core/iwasm/common/wasm_runtime_common.h

```diff
@@ -264,7 +271,92 @@ STORE_U16(void *addr, uint16_t value)
     ((uint8_t *)(addr))[0] = u.u8[0];
     ((uint8_t *)(addr))[1] = u.u8[1];
 }
+
+#define STORE_V128(addr, value)                     \
+    do {                                            \
+        uintptr_t addr_ = (uintptr_t)(addr);        \
+        union {                                     \
+            V128 val;                               \
+            uint64 u64[2];                          \
+            uint32 u32[4];                          \
+            uint16 u16[8];                          \
+            uint8 u8[16];                           \
+        } u;                                        \
+        if ((addr_ & (uintptr_t)15) == 0)           \
+            *(V128 *)(addr) = (V128)(value);        \
+        else {                                      \
+            u.val = (V128)(value);                  \
+            if ((addr_ & (uintptr_t)7) == 0) {      \
+                ((uint64 *)(addr))[0] = u.u64[0];   \
+                ((uint64 *)(addr))[1] = u.u64[1];   \
+            }                                       \
+            else if ((addr_ & (uintptr_t)3) == 0) { \
```

Comment:
If we always only access the v128 operands in the interpreter's stack, since the stack operand must be stored in 4-byte aligned address (only i32/i64/f32/f64/v128 operands), here we can simplify the code to be like:
```C
    else {
        bh_assert((addr_ & (uintptr_t)3) == 0);
        ((uint32 *)(addr))[0] = u.u32[0];
        ((uint32 *)(addr))[1] = u.u32[1];
        ((uint32 *)(addr))[2] = u.u32[2];
        ((uint32 *)(addr))[3] = u.u32[3];
    }
    => the last two else statements are removed.
```
or the interpreter's code may be very large when the macros are expanded.